lock/src/test/java/org/infinispan/lock/ClusteredLockTest.java

```diff
@@ -161,38 +162,71 @@ public void testTryLockWithTimeoutWithCountersInParallel() throws Throwable {
       assertEquals(3, counter.get());
    }
 
+   @Test
+   public void testTryLockWithTimeoutWithCountersInParallel() throws Throwable {
+      AtomicInteger counter = new AtomicInteger();
+
+      ClusteredLock lock0 = clusteredLockManager(0).get(LOCK_NAME);
+      ClusteredLock lock1 = clusteredLockManager(1).get(LOCK_NAME);
+      ClusteredLock lock2 = clusteredLockManager(2).get(LOCK_NAME);
+
+      await(CompletableFuture.allOf(
```

Comment:
I prefer waiting on each future separately instead of waiting on `allOf`, but I see the existing tests already use `allOf` quite a bit.

Reply:
done